kubernetes/README.md

```diff
@@ -0,0 +1,99 @@
+Deploying to a Kubernetes cluster involves several steps. Below is a step-by-step guide that assumes you have a Kubernetes cluster already set up and configured, and you have `kubectl` installed and configured to communicate with your cluster. This guide also assumes you have already built and pushed your Docker images to a container registry that your Kubernetes cluster can access.
```

Comment:
It's not clear to me why we would want both RAW kubernetes YAML and a HELM chart.  Can you share more about you're thinking here?

Reply:
For those of us that use Kustomize or other systems we don't want helm charts.

Reply:
You can do that. But if you are a user that does not in fact use helm, should they have to install helm cli locally to be able to generate manifests so they can avoid using helm?

In scenarios where gitops is the primary deployment mechanisms the barrier of entry will be a lot lower if the manifests is provided as well depending on the tooling used.